changes/ce/fix-10449.en.md

```diff
@@ -0,0 +1,2 @@
+Validate the ssl_options and header configurations when creating authentication http (`authn_http`).
+Prior to this, incorrect ssl_options configuration could result in successful creation but the entire authn being unusable.
```

Comment:
Will this be better? In fact, the name of this configuration is `ssl` instead of `ssl_option`?
```suggestion
Prior to this, incorrect `ssl` configuration could result in successful creation but the entire authn being unusable.
```